about/credits/index.html

```diff
@@ -1,9 +1,10 @@
 <!DOCTYPE html>
 <html lang="en-US">
 
-<head><meta name="twitter:site" content="@AytchSoftware"><meta property="og:image" content="https://h-projects.github.io/assets/images/brand/loqo.png"><meta property="og:site_name" content="Aytch Software"><meta property="og:type" content="website" />
-	<!-- import over logic -->
-	<script src="../assets/index.js"></script>
+<head>
+	<!-- js -->
+	<script src="/assets/index.js"></script>
+	<!-- ga -->
 	<script async src="https://www.googletagmanager.com/gtag/js?id=UA-169183883-1"></script>
```

Comment:
Why do we need Google analytics.
It's useless for anyone and spies on people for no reason.

Reply:
Google Analytics does actually help with SEO from what I understand, in that it sells all that data to google /s.

Reply:
It doesn't if you don't use these analitycs for improving SEO.

Reply:
SEO is fucked because ASTK/CNAME.

Reply:
SEO has nothing to do with CNAME.

Reply:
Site down = Shit SEO.